index.js

```diff
@@ -91,13 +91,15 @@ export default class extends Component {
     Animated.timing(this.scaleFactor, {
       toValue: 1,
       duration: SCALE_UP_DURAITON,
+      useNativeDriver: false,
```

Comment:
I'm not sure whether the value of useNativeDriver should be true or false, so I followed what was already set for Animated.event.